apps/judicial-system/web/src/routes/Court/Indictments/ReceptionAndAssignment/ReceptionAndAssignment.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useContext, useState } from 'react'
```

Comment:
Quick question about this file. The Reception and Assignment screens are identical to each other in all case types, except for where the next button routes. Do you think we should use one component for all these pages? @Arnarkari93

Reply:
There are two ways I would think are acceptable. The first is to have a shared component for all these pages as you mention. The other is to move shared code to custom hooks and components and have three different setup pages.

Reply:
I am more leaning toward the first option as this page is not complex.